version/version_test.go

```diff
@@ -0,0 +1,25 @@
+package version_test
+
+import (
+	"fmt"
+	"runtime"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"go.dagger.io/dagger/version"
+)
+
+var _ = Describe("Version", func() {
```

Comment:
We use `stretchr/testify` across the code base (see `context_test.go`, `git_test.go`, etc)

Reply:
OK, I will switch to this one.